src/types.ts

```diff
@@ -1,7 +1,105 @@
+import * as pulumi from '@pulumi/pulumi';
+import { Stack, StackProps } from 'aws-cdk-lib/core';
+import { ResourceMapping } from './interop';
+/**
+ * Options specific to the Stack component.
+ */
+export interface StackOptions extends pulumi.ComponentResourceOptions {
```

Comment:
This is part of public API, right?

Reply:
Yep, I just updated the exports to only include this since it is the only thing we want  exported from types.